src/lib.rs

```diff
@@ -443,10 +450,15 @@ fn start_flush_cycle(wbuf: &Arc<Mutex<Outbound>>) -> usize {
     w.writer.buffer().len()
 }
 
+/// A `Message` that has been published to a NATS `Subject`.
 #[derive(Debug)]
 pub struct Message {
+    /// The NATS `Subject` that this `Message` has been published to.
     pub subject: String,
+    /// The optional unique reply inbox that may be used for
```

Comment:
Does not have to be unique, could be anything you want.

Reply:
Thanks! I misinterpreted the docs at https://docs.nats.io/developing-with-nats/sending/replyto but looking back at it the wording is not very ambiguous that it CAN be unique instead of MUST be unique.

Reply:
Yep!